Makefile

```diff
@@ -101,3 +105,4 @@ ifndef HAS_GOX
 	go get -u github.com/mitchellh/gox
 endif
 	glide install
+	@echo Please run pip install -r expansion/requirements.txt. You might need sudo.
```

Comment:
`pip install --user -r expansion/requirements.txt` might be what you're looking for.

```
  --user                      Install to the Python user install directory for your platform. Typically ~/.local/, or %APPDATA%\Python on Windows. (See the Python
                              documentation for site.USER_BASE for full details.)
```